templates/which-app/decorators/decorateGeneral.js

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2021 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+export default function decorateGeneral() {
+  document.title = 'Which App Should I Use? | Adobe Creative Cloud';
```

Comment:
Is this always be English? I am asking this because the URL includes `en` and it seems there could be other locale someday. In that case this can't be hard coded title.

Reply:
All hard-coded language-specific content has been fixed.